backend/src/main/java/com/redhat/cloud/notifications/OBTest.java

```diff
@@ -0,0 +1,246 @@
+package com.redhat.cloud.notifications;
+
+import com.redhat.cloud.notifications.db.repositories.TemplateRepository;
+import com.redhat.cloud.notifications.models.IntegrationTemplate;
+import com.redhat.cloud.notifications.openbridge.Bridge;
+import com.redhat.cloud.notifications.openbridge.BridgeApiService;
+import com.redhat.cloud.notifications.openbridge.BridgeAuth;
+import com.redhat.cloud.notifications.openbridge.BridgeEventService;
+import com.redhat.cloud.notifications.openbridge.Processor;
+import io.vertx.core.json.JsonObject;
+import org.eclipse.microprofile.rest.client.RestClientBuilder;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+import javax.inject.Inject;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.WebApplicationException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Helper during development
+ */
+@Path("/internal/ob-test")
+public class OBTest {
```

Comment:
Check if that works:
```suggestion
@IfBuildProfile("dev")
public class OBTest {
```